src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -235,7 +226,7 @@ def get(self, request, organization):
             if search_filter.key.name == "status"
         ]
         if status and status[0].value.raw_value == GroupStatus.UNRESOLVED:
-            context = [r for r in context if r["status"] == "unresolved"]
+            context = [r for r in context if "status" not in r or r["status"] == "unresolved"]
```

Comment:
Why do we need this?

Reply:
If we call this endpoint with `collapse=base`, status won't be included and this would cause errors